src/components/Badge/Badge.scss

```diff
@@ -18,7 +18,7 @@ $pip-spacing: ($height - $pip-size) / 2;
   display: inline-flex;
   align-items: center;
   padding: 0 $horizontal-padding;
-  background-color: var(--p-neutral, color('sky'));
+  background-color: var(--p-neutral-action-disabled, color('sky'));
```

Comment:
Until we have a better variant for this purpose, I've gone ahead and used the neutral disabled action.